src/libraries/Microsoft.PowerFx.Core/Functions/IHasUnsupportedFunctions.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    internal interface IHasUnsupportedFunctions
+    {
+        bool IsDeprecated { get; }
+
+        bool IsSupported { get; }
+
+        string NotSupportedReason { get; }
```

Comment:
>string [](http://example.com/codeflow?start=8&length=6)

Is this is a message we show users in intellisense?
Does that mean we need to worry about the locale?

Reply:
I can add a comment to mention this (internal use only)